cluster/manifests/deletions.yaml

```diff
@@ -40,6 +40,16 @@ pre_apply:
   namespace: kube-system
   kind: Deployment
 
+{{ if eq .ConfigItems.fabric_gateway_controller_mode "off" }}
+- name: fabric-gateway-controller
+  namespace: kube-system
+  kind: Deployment
+- labels:
+    fabric-gateway: "true"
+  kind: RouteGroup
+{{ end }}
```

Comment:
Should this include RBAC deletions as well? The problem is that it takes some time to scale down deployment on deletion which prevents deletion of a ServiceAccount that is in use.

Reply:
Decided to avoid deletions altogether